### DISCUSSION TITLE
Docker compose - does not install

### DISCUSSION THREAD

**Discussion by Emanuele-DV** [indomus/forum#1731] [HUB: Home Assistant]
Hello everyone

I followed the Docker guide on Linux Debian

arrived at the command

- sudo python3 -m pip install docker-compose

the answer was ... "No Module named pip" 

while the command

- .docker run hello-world

he gave replied

- Hello from Docker!

- This message shows that your installation appears to be working correctly

How can I solve ... Thanks ... so I continue with the rest of the installation

**Comment by Scialla** [comment]
sembra strano tu non abbia pip installato su python... prova così:
`sudo apt-get install python3-pip`

**Comment by Emanuele-DV** [comment]
Buonasera 

sono stato 3 ore a tentare niente ... al 99% si ferma e da questa risposta 

Scaricamento di:1 http://security.debian.org/debian-security bullseye-security/main amd64 dpkg-dev all 1.20.10 [2.306 kB]
Errore:1 http://security.debian.org/debian-security bullseye-security/main amd64 dpkg-dev all 1.20.10
  Connessione terminata [IP: 199.232.82.132 80]
E: Impossibile recuperare http://security.debian.org/debian-security/pool/updates/main/d/dpkg/dpkg-dev_1.20.10_all.deb  Connessione terminata [IP: 199.232.82.132 80]
E: Impossibile scaricare alcuni pacchetti. Potrebbe essere utile eseguire "apt-get update" o provare l'opzione "--fix-missing".

come posso fare .... Grazie 

ps dal pc lo ha scaricato .... dove lo devo mettere su linux per farlo vedere insieme al resto ???
